packages/block-library/src/button/edit.js

```diff
@@ -115,34 +115,29 @@ function useEnter( props ) {
 }
 
 function WidthPanel( { selectedWidth, setAttributes } ) {
-	function handleChange( newWidth ) {
-		// Check if we are toggling the width off
-		const width = selectedWidth === newWidth ? undefined : newWidth;
-
-		// Update attributes.
-		setAttributes( { width } );
-	}
-
 	return (
 		<PanelBody title={ __( 'Settings' ) }>
-			<ButtonGroup aria-label={ __( 'Button width' ) }>
+			<ToggleGroupControl
```

Comment:
I feel like we should add the `isBlock` prop here:

- it will make the control full-width
- it will show a border

Reply:
Makes sense. I'll add the prop.

What's the reason for having border style connected to the `isBlock` prop?

Reply:
I believe it was decided together with design folks given the specific constraints that we faced on the component — but I'll let Lena clarity if necessary.

Reply:
Yes, the relationship between the props and the patterns are not very clear at the moment. The three "valid" patterns we can have are:

- Text options + `isBlock`
- Icon options
- Icon options + `isDeselectable`

I'll make a note about this for when we stabilize this component (#61067).